### PR TITLE
[ZEPPELIN-1462] Zeppelin-Web Good Practices #3

### PR DESCRIPTION
**Pull request by corneadoug**
### What is this PR for?
This Good Practice Guide will focus on how to write a controller following the controller as vm concept, and how to provide a nice separation of the code.

### What type of PR is it?
Documentation

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-1462

### How should this be tested?
Check the `.md` file, or build the website to see the full rendering

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by corneadoug** [comment]
@AhyoungRyu Since you always find a few mistakes, can you make a quick spell check like always? :)

File: contribution/zeppelinweb/goodPracticeGuide03.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# How to Write a Controller
+
+<br/>
+Our main rule regarding writing a controller, is to use the `Controller as vm` style.
+
+However we also have a few other style guidelines.
```
Comment (prabhjyotsingh) [inline contribution/zeppelinweb/goodPracticeGuide03.md]: However,

Comment (corneadoug) [inline contribution/zeppelinweb/goodPracticeGuide03.md]: @prabhjyotsingh You really like that comma :)

File: contribution/zeppelinweb/goodPracticeGuide03.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+You can see below a full example of what the controller would look like.
+
```
Comment (AhyoungRyu) [inline contribution/zeppelinweb/goodPracticeGuide03.md]: Do we need additional new lines? How about putting those three lines in one line? I mean

```
Our main rule regarding writing a controller, is to use the `Controller as vm` style.
However we also have a few other style guidelines. You can see below a full example of what the controller would look like.
```

Comment (corneadoug) [inline contribution/zeppelinweb/goodPracticeGuide03.md]: I would have prefered those lines to be one under another without any additional spacing, but If I don't have that new line, the `You can` goes straight on the right side of the previous sentence

Comment (corneadoug) [inline contribution/zeppelinweb/goodPracticeGuide03.md]: Eventually I could group the first two sentences:

```
Our main rule regarding writing a controller, is to use the `Controller as vm` style, however, we also have a few other style guidelines.
```

**Comment by corneadoug** [comment]
Changes done.
Merging if there is no other discussions

**Pull request closed without merging**